releasenotes/notes/qrao-89d5ff1d2927de64.yaml

```diff
@@ -0,0 +1,52 @@
+---
+features:
+  - |
+    Added a new optimization algorithm, :class:`~.QuantumRandomAccessOptimizer`. This approach
+    incorporates Quantum Random Access Codes (QRACs) as a tool to encode multiple classical binary
+    variables into a single qubit, thereby saving quantum resources and enabling exploration of
+    larger problem instances on a quantum computer. The encodings produce a local quantum
+    Hamiltonian whose ground state can be approximated with standard algorithms such as VQE and
+    QAOA, and then rounded to yield approximation solutions of the original problem.
+
```

Comment:
As the example below shows both normal and relaxed solves, should we add some text here to describe these a bit, mainly the relaxed aspect since thats involves more so readers have a bit better idea when they are looking at the example below which shows both.